lobsterpy/plotting/__init__.py

```diff
@@ -226,3 +236,427 @@ def _broaden(energies: np.ndarray, population: np.ndarray, sigma=None, cutoff=4.
+                    + ": "
+                    + label_srt[0].strip("123456789")
+                    + "-"
+                    + label_srt[1].strip("123456789")
+                )
+                labels[inx].append(bond_info.labels[ixx])
+
+        # create a dict seperating the unique atom pairs for each site and corresponding cohp bond label
+        plot_data = {}
+        for indx, atom_pairs in enumerate(bonds):
+            search_items = set(atom_pairs)
+            for item in search_items:
+                indices = [i for i, x in enumerate(atom_pairs) if x == item]
+                filtered_bond_label_list = [labels[indx][i] for i in indices]
+                plot_data.update({item: filtered_bond_label_list})
+
+        if "All" not in self._cohps:
+            self._cohps["All"] = {}
+
+        if label_resolved:
+            for bond_key, labels in plot_data.items():
+                count = len(labels)
+                label_with_count = self._insert_number_of_bonds_in_label(
+                    label=bond_key, character=":", number_of_bonds=count
+                )
+                self._cohps[label_with_count + suffix] = {}
+                for label in labels:
+                    cohp = complete_cohp.get_cohp_by_label(label)
+                    energies = (
+                        cohp.energies - cohp.efermi
+                        if self.zero_at_efermi
+                        else cohp.energies
+                    )
+                    populations = cohp.get_cohp()
+                    int_populations = cohp.get_icohp()
+                    outer_key = label_with_count + suffix
+                    struct = analyse.structure
+                    atom_pairs = []
+                    for site in complete_cohp.bonds[label]["sites"]:
+                        atom = site.species_string + str(struct.sites.index(site) + 1)
+                        atom_pairs.append(atom)
+                    key = "{}: {} ({} \u00c5)".format(
+                        label,
+                        "-".join(atom_pairs),
+                        str(round(complete_cohp.bonds[label]["length"], 2)),
+                    )
+                    self._cohps[outer_key].update(
+                        {
+                            key: {
+                                "energies": energies,
+                                "COHP": populations,
+                                "ICOHP": int_populations,
+                                "efermi": cohp.efermi,
+                            }
+                        }
+                    )
+
+                    # alpha = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                    # new_label = (
+                    #    bond_key.split(":")[0].strip(alpha)
+                    #    + "_"
+                    #    + bond_key.split(":")[1].strip()
+                    #    + ": "
+                    #    + label
+                    # )
```

Comment:
Looks like this commented-out code can be cleaned up

Reply:
Thanks! Deleted the comments.